packages/flutter_test/lib/src/_matchers_web.dart

```diff
@@ -90,7 +89,7 @@ RenderObject _findRepaintBoundary(Element element) {
 }
 
 void _renderElement(ui.Window window, RenderObject renderObject) {
-  final Layer layer = renderObject.debugLayer;
+  final Layer layer = renderObject.debugLayer!;
```

Comment:
similar here; what if someone ends up calling this before layout/paint have run? you can change the phase in a test.